dtests/src/test/scala/io/snappydata/hydra/northwind/NWTestUtil.scala

```diff
@@ -103,6 +68,22 @@ object NWTestUtil {
     Row.fromSeq(md)
   }
 
+  def writeToFile(df: DataFrame, dest: String, snc: SnappyContext): Unit = {
+    import snc.implicits._
+    df.map(dataTypeConverter)(RowEncoder(df.schema))
+      .map(row => {
+        val sb = new StringBuilder
+        row.toSeq.foreach(e => {
+          if (e != null)
+            sb.append(e.toString).append(",")
+          else
+            sb.append("NULL").append(",")
+        })
+        sb.toString()
+      }).write.format("org.apache.spark.sql.execution.datasources.csv.CSVFileFormat").option(
```

Comment:
@swatisawant @SachinJanani Note that there are two main changes for dunits: a) repartition(1).sortWithinPartitions helps run the query in parallel as we also discussed in the presentation so the same should be used in hydra, b) writing directly instead of CSV writer as you mention